src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -1253,3 +1254,101 @@ def get_parent_proc_name():
         parent_proc_name = _get_parent_proc_name()
         setattr(get_parent_proc_name, "return_value", parent_proc_name)
     return getattr(get_parent_proc_name, "return_value")
+
+
+def log_cmd_history(command, args):
```

Comment:
Consider putting this function in a separate file, because `util.py` is already too big containing too many functions with different purposes.